### PR TITLE
Add output of categories, sticky, author bio, single pagination styling, comments styling

### LINKED ISSUE #321: Add author Bio below single posts
In the design there is an author bio with avatar below the meta information and above the navigation: 

author

### LINKED ISSUE #324: Update separators for next and previous post navigation
The style for the separators above and below the next and previous post navigation needs to be updated.
The separator uses the same design as the **_wide line separator block._**

author

### PR DESCRIPTION
**Pull request by andersnoren**
Adds missing elements and styles to `content.php` and template parts used by `content.php`.

- Update separators for next and previous post navigation (fixes #324)
- Add author bio below single posts (fixes #321)
- Updates the comments structure and styles to bring it closer to the design
- Adds output of a "Sticky post" meta item to the top meta position
- Adds output of categories to the top of the entry header
- Updates the post intro text to be closer to the design

### CODE REVIEW

File: comments.php [hunk 1]
```diff
@@ -20,60 +20,86 @@
...
 
-			<h3 class="comment-reply-title"><?php echo esc_html( $comments_title ); ?></h3>
+			<h2 class="comment-reply-title">
+			<?php
+			if ( comments_open() ) {
+				if ( ! have_comments() ) {
+					esc_html_e( 'Leave a comment', 'twentynineteen' );
+				} elseif ( '1' === $comments_number ) {
+					/* translators: %s: post title */
+					printf( esc_html_x( 'One reply on &ldquo;%s&rdquo;', 'comments title', 'twentynineteen' ), esc_html( get_the_title() ) );
```
Comment (carolinan) [inline comments.php]: The text domain needs to be changed to twentytwenty.

File: template-parts/entry-author-bio.php [hunk 2]
```diff
@@ -0,0 +1,33 @@
...
+if ( (bool) get_the_author_meta( 'description' ) ) : ?>
+<div class="author-bio">
+		<h2 class="author-title heading-size-4">
+			<div class="author-avatar vcard">
+				<?php echo get_avatar( get_the_author_meta( 'ID' ), 160 ); ?>
+			</div>
+			<span class="author-name">
+				<?php
+				printf(
+					/* Translators: %s: post author */
```
Comment (carolinan) [inline template-parts/entry-author-bio.php]: This content "By ...." is not inside an attribute, replace esc_attr with esc_html.

File: template-parts/entry-author-bio.php [hunk 3]
```diff
@@ -0,0 +1,33 @@
...
+					esc_attr( 'By %s', 'twentytwenty' ),
+					esc_html( get_the_author() )
+				);
+				?>
+			</span>
+		</h2>
+		<p class="author-description">
+			<?php the_author_meta( 'description' ); ?>
+			<a class="author-link" href="<?php echo esc_url( get_author_posts_url( get_the_author_meta( 'ID' ) ) ); ?>" rel="author">
+				<?php esc_attr_e( 'View Archive &rarr;', 'twentytwenty' ); ?>
```
Comment (carolinan) [inline template-parts/entry-author-bio.php]: This content is not inside an attribute, replace esc_attr_e with esc_html_e.

**Comment by andersnoren** [comment]
Thanks @carolinan – updated.

**Pull request merged**